jumpscale/tools/restic/restic.py

```diff
@@ -211,16 +211,24 @@ def list_snapshots(self, tags=None, last=False, path=None):
         proc = self._run_cmd(cmd)
         return json.loads(proc.stdout)
 
-    def forget(self, keep_last=10, prune=True):
+    def forget(self, keep_last=10, prune=True, snapshots=[], tags=[]):
```

Comment:
please don't use list as arguments in functions/methods definitions, as python default arguments are only evaluated `once`